### DISCUSSION TITLE
Upgrading msticpy notice on README

### DISCUSSION THREAD

**Discussion by titanicshark** [microsoft/msticpy#272] [Ideas]
Hi. 

I thought it might be a nice idea to have a notice on the main branch README letting people know that they can upgrade to the latest msticpy version with the command: pip install --upgrade msticpy and for users to store their own copy of msticpyconfig.yaml elsewhere on their machine, referenced in an environment variable so they don't lose their config when upgrading.

**Comment by petebryan** [comment]
Nice idea! Do you want to make that change and raise a PR for it?

**Comment by titanicshark** [comment]
Sure. By the way, are you able to tell me if I need an API key to use the
OPR Open Page Rank threat intelligence provider with msticpy? If so, where
can I get said key?

Regards,

Daniel Cockburn-Evans

**Comment by ianhelle** [comment]
Hi Daniel, You should be able to sign up here. https://www.domcop.com/openpagerank/auth/signup
